Gemfile-rails-dependencies

```diff
@@ -31,3 +31,6 @@ else
     gem "puma"
   end
 end
+
+gem 'activerecord-jdbcsqlite3-adapter', git: 'https://github.com/jruby/activerecord-jdbc-adapter', platforms: [:jruby]
```

Comment:
Don't we want to specify a version?

Reply:
I worked with @samphippen on the JRuby support at Ruby Kaigi and it appears there are issues on Travis that aren't likely to be fixed soon. This was the best we could come up with 😠

Reply:
Thanks again :)